examples/proxy-server/src/main/resources/index.html

```diff
@@ -42,7 +46,8 @@ <h1 align="center">Armeria animation!</h1>
     });
 </script>
 <body>
-  <div class="description">Press F5 to access the different backend!</div>
+  <h1 align="center">Armeria animation!</h1>
+  <div class="description">Press F5 to access the different backend.</div>
```

Comment:
nit: How about changing from `Press F5` to `Refresh your browser`? Because F5 doesn't work on OSX.